browser-config.js

```diff
@@ -24,7 +24,6 @@ module.exports = {
 		"no-duplicate-imports": 2,
 		"no-restricted-syntax": ["error", "CatchClause[param=null]"],
 		"no-useless-constructor": 2,
-		"no-var": 2,
 		"prefer-arrow-callback": 2,
 		"prefer-const": 2,
```

Comment:
I think we should move `prefer-const` over as well?

> Require const declarations for variables that are never reassigned after declared

Reply:
It's there - want me to remove this one?

Reply:
Oh right -- yeah it definitely doesn't need to be in both places.

Reply:
88626dd99396a0f6c6bd1341a3476ce55e090070